modules/slack/resources/schema.yaml

```diff
@@ -0,0 +1,47 @@
+title: Schema for Jenkins Slack Notification plugin configuration
+type: object
+additionalProperties: false
+
+properties:
+  team_subdomain:
```

Comment:
I'd propose to replace this with just slack url. There could be on-premise installations in future. And it's just easier to copypaste url than try to understand, which part of it is needed